frontend_tests/casper_tests/00-realm-creation.js

```diff
@@ -1,7 +1,9 @@
+/* global domain */
+
 var common = require('../casper_lib/common.js').common;
 
 var email = 'alice@test.example.com';
-var domain = 'test.example.com';
+window.domain = 'test.example.com';
```

Comment:
Oh, sorry, I think the test is failed because I remove this line, then I add this again and success. Just now I test it when the line just removed and success, look like a intermittent fails.

I'll remove this line, thanks for noticing